core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -537,6 +539,71 @@ public T proxyDetector(ProxyDetector proxyDetector) {
     throw new UnsupportedOperationException();
   }
 
+  /**
+   * Provides a service config to the channel.
```

Comment:
Also document the cases this default config will be used. My understanding of the [spec](https://github.com/grpc/proposal/blob/master/A21-service-config-error-handling.md) is the default config is used only if the Channel has not seen any config from NameResolver, including the case config lookup is disabled.

Reply:
Done.